src/Patch/Document.js

```diff
@@ -37,7 +37,7 @@ export default function(internals) {
      * @return {!Node}
      */
     function(node, deep) {
-      const clone = Native.Document_importNode.call(this, node, deep);
+      const clone = /** @type {!Node} */ (Native.Document_importNode.call(this, node, !!deep));
```

Comment:
Could you comment that this `!!` (here and in `cloneNode` is because of a closure bug? (AFAICT, it thinks this isn't optional.) Also, I why it doesn't think `importNode` would return a `Node`? Is there a case where that can happen, excluding when it throws?

Reply:
Also per MDN (https://developer.mozilla.org/en-US/docs/Web/API/Document/importNode):

> In the DOM4 specification, deep is listed as an optional argument. If omitted, the method acts as if the value of deep was true, defaulting to using deep cloning as the default behavior. To create a shallow clone, deep must be set to false.

> This behavior has been changed in the latest spec, and if omitted, the method will act as if the value of deep was false. Though It's still optional, you should always provide the deep argument both for backward and forward compatibility. With Gecko 28.0 (Firefox 28 / Thunderbird 28 / SeaMonkey 2.25 / Firefox OS 1.3), the console warned developers not to omit the argument. Starting with Gecko 29.0 (Firefox 29 / Thunderbird 29 / SeaMonkey 2.26)), a shallow clone is defaulted instead of a deep clone.